flash_cards/models.py

```diff
@@ -1,3 +1,21 @@
 from django.db import models
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+# from django.utils.text import slugify
 
 # Create your models here.
+class FlashCard(models.Model):
+    title = models.CharField(max_length=50)
+    description = models.TextField(max_length=1000, null=True, blank=True)
+    deck = models.ForeignKey('Deck', on_delete=models.CASCADE)
+
+
+    def __str__(self):
+        return self.title
+
+class Deck(models.Model):
+    title = models.CharField(max_length=50)
+    created_at = models.DateTimeField(auto_now_add=True)
+
+    def __str__(self):
+        return self.name
```

Comment:
```suggestion
        return self.name
        
        def __repr__(self):
             return f"<Deck name={self.name}>"
```
This allows us to call the deck by name in the future, and sets us up if we want to sugify them as well

Reply:
Really good job, Tyler! I'll let @JDitto95 take a look, then we should be able to merge!